### PR TITLE
Adds NT Instructor job for Mentors

### PR DESCRIPTION
**Pull request by Kyep**
## What Does This PR Do
Adds a new job, "NT Instructor", based on a similar job on CM: https://cm-ss13.com/wiki/Senior_Enlisted_Adviser
This is a mentor/admin exclusive job that lets mentors teach new players ICly.
They are not included in job prefs, but you can latejoin as one if you're a mentor/admin.
Their radio has all channels, similar to NTR.
Their access is based on NT Rep, except that they do NOT get access to personal lockers, crematorium, vault, keycard auth, gateway, NTR office. So their access is between that of an IAA and the NT Rep.

## Why It's Good For The Game
Allows mentors to teach new players ICly.

## Images of changes
nt_instructor
ntinstructor_sop

## Changelog
:cl: Kyep
add: Added NT Instructor, a mentor/admin exclusive job aimed at teaching new players basic game mechanics.
/:cl:

### CODE REVIEW

**Comment by TDSSS** [comment]
Mindshield and dust implant, I see. They're going to be pretty damn strong against 'all crew can fight these' antags.

Also, why only one slot for this?

**Comment by Kyep** [comment]
@TDSSS 
They won't be allowed to fight antags. They exist to teach, not fight.
I want to see how they go with only one job slot, before I consider whether more slots could work. I'm probably not going to give them more than one slot.

**Comment by TDSSS** [comment]
To clarify, they are not allowed to fight even wizards, nukies, blobs, terror spiders and co?

**Comment by Kyep** [comment]
Correct.
They are to stay out of all crew/v/antag battles. They're just there to teach.

**Comment by TDSSS** [comment]
Hmm, maybe you could give them pacifism when they spawn in, then?

**Comment by Kyep** [comment]
I thought about it, but I worried that'd prevent them from even defending themselves if they were attacked by a NPC, such as a spider.

**Comment by Ty-Omaha** [comment]
My concerns with this:

- I worry some people will use this to act as a mini CC / Navy Officer, especially with the command comms.
- They should not be giving their 2 cents on anything going on unless explicitly asked.
- It needs to be made clear they have no actual authority, and not even in an official advisor capacity such as the NTR.
- It should only be used if a player really needs help like a new person learning atmos. I don't want to see these very special roles just running around station doing nothing or talking at the bar with their fancy outfit and title.

Apart from that I think this is an alright idea.

**Comment by Pixelzilla** [comment]
Why exactly do we need this? What prevents new players from seeking help from their co-workers, a pAI or anyone else actively in the round already? It just seems like a weird level of snowflakey CC characters that really shouldn't be there with immunity on the station. I cant say I've ever really seen a situation where a player really needs help/wants to learn and absolutely no one on station is willing to help them. This feels like it is catering more towards the person assuming the role than the person needing help? Thats just a lot of stuff, special title and recognition for a mentor and it almost feels like it puts the wrong sort of incentive to apply to being one.

**Comment by Kyep** [comment]
Do we 'need' mimes, librarians, virologists, or most other jobs on the station? No, but the station is better for having them. 
They're no more "CC characters" than the existing blueshield or NT Rep are, and those jobs too perform a useful function.
You say you've never seen situations like this, but I have. I experienced them myself when I was new back in the day. Plus, just because someone wants to help you doesn't mean they can. Mentors have enough game knowledge that they actually can, most of the time.
The amount of stuff they get is quite limited. They get less access than the NT Rep, for example. They don't get an office, either. If you think this job is shiny, then you could just play NT Rep instead, which has more stuff, more access, and an office. This job is really not that shiny - once the initial wave of curious mentors wears off, I only expect to see it when someone genuinely wants to teach.

**Comment by farie82** [comment]
All for this change. As a mentor we don't have aghost or such. We have to go on what the person asking the question is telling us. Which is usually limited because they don't have experience in the thing they are trying to do.

Asking your coworkers will still be done and should be done even if this is implemented. But have you tried to ask your coworkers as of late? 9/10 times they are busy since they have a huge playerbase to uphold. For example doctors? Round start them have some time till the first wave of people hit who are wounded due to random fights. Then another wave from the first antag confrontation. Etc.

Having this whitelisted to only mentors and other players who are trusted is a good idea. Mentors already have tools to "powergame/metagame" but we don't use them for that. It's a trust thing.

I'll fully review the code once I have time to see if I can spot any issues with the design and the code itself.

All in all I believe this will add to the game simply because the person playing this can fully teach somebody a job if they are asked to. This is not something we can do as a mentor currently due to the communication limitations

**Comment by procdrone** [comment]
@Pixelzilla Other players or cooworkers aren't really required or expected to babysit new players, some might want to play their rounds without such. Not to mention job requirements. 

You would be suprised how many people DONT want to teach newbies and just do their job.

The Instructor is removed from the usual round flow so he can easily spend the whole shift on teaching someone, and noone will be worse off.

**Comment by AffectedArc07** [comment]
The good thing with mentors is that you have a group of people using an OOC system (admin PM) so that not only can you get instant responses without an instructor having to walk to you, because it’s OOC you can ask antag or meta questions. If we really are medium RP, we’re not going to have a CC official telling a crew member “oh yeah just set this to your ringtone for traitor gear”.

Put short, I don’t like this

File: code/modules/mob/new_player/new_player.dm [hunk 1]
```diff
@@ -261,6 +261,7 @@
 	if(!is_job_whitelisted(src, rank))	 return 0
 	if(!job.player_old_enough(client))	return 0
 	if(job.admin_only && !(check_rights(R_EVENT, 0))) return 0
+	if(job.mentor_only && !(check_rights(R_ADMIN|R_MOD|R_MENTOR, 0))) return 0
```
Comment (AffectedArc07) [inline code/modules/mob/new_player/new_player.dm:264]: The R_MOD right is a carryover from baystation and (afaik) has never been used, nor is it planned to be. Don’t know if this is necessary or not.

**Comment by Fox-McCloud** [comment]
Very strongly opposed to this. Paradise has largely gotten along fine without such a role and I don't think it'll open up anything particularly good without lots of additional problems.

The more a role requires OOC rules to remain balanced and not break things the more we should question how well designed it is to begin with. 

This has all the hallmarks of that in addition to a lot of other questionable changes.

The role is essentially immune from being an antag target, which, in and of itself is quite questionable. They're supposedly, by OOC rules, not supposed to interfere, yet they have a radio headset that gives them access to every channel and a telebaton which can really only be utilized for self-defense or going after antags. To even remotely suggest that players aren't going to, one day, start utilizing (intentionally or unintentionally) this to interfere with the round is naive, in my opinion. 

The role itself is likely to also devolve into players being viewed as "jerks" if they so much as kill one of these guys "why kill them, they can't be antag anyway; it's just being a jerk--it's also rude to go after people teaching". Fundamentally, I dislike this because SS13 is very much a roleplaying game with chaos that embodies "murphy's law" in action---a player should never grow entitled to being able to learn, for sure, X in a particular round or survive it, or for things to not go sideways---they should be well acquainted, from day 1, that dying horrendously and unfairly is very much part of the game.

On boring shifts, the role is also just plain going to get abused--if no one is need of instruction, it's pretty much jsut free game to wander around to any department you want doing....whatever you want. This doesn't even have to be something maliciously done on the player's part; it's going to happen with the role on its own out of sheer boredom. You know how assistants can be when they're bored (yes, that goes for admins and mentors too!); now imagine one with a telebaton, shades, and basic access to everything.

From a roleplay perspective, this makes utterly zero sense; CentComm isn't going to send one of their very own to the station just to teach an entry level employee how to work a machine or now something functions, or that "dat phrase is tater tot language!" (or whatever else have you). Realistically, again, from a roleplay perspective, how you learn is far more realistic---you're going to pick up things from the players who you're around who are in your job already, or someone is going to transfer to a department to help a newbie.

Big thumbs down for me, for design, purpose, problems it will cause, and altering roleplay to be less emergent and ICly a bit strange.

👎

**Comment by Pixelzilla** [comment]
@Kyep 
It looks like it has all access, a radio with all channels, a full toolbelt, a telescopic baton, a pinpointer with direct instructions to not interfere with antags and somehow no one else thinks this is a loot pinata? Unless I'm reading the central.dm entirely wrong then that's what seems to be proposed?

The entire point I'm trying to make here is that if a mentor wants to help someone ICly what exactly prevents that same mentor from actually doing so as a normal character with no bells and whistles? Why do we need a special role with a special title to do something that you can actively do right now without this PR. Nothing stops anyone from helping someone out, you don't even need to be a mentor to be a helpful person.

**Comment by Quantum-M** [comment]
Why not make it, instead of a humanoid with a Central Command like getup, some sort of hologram that can phase through anything,cannot be hurt, can speak common, and has hands to pick up stuff?

That way there's no worry in regards of the loot, and they can be easily move around without worry of being attacked. And given that it's only intended for mentors, there shouldn't be as much worry of the it being abused as they are supposed to be trustable.

**Comment by procdrone** [comment]
@Fox-McCloud 

With what you're saying, we should remove all mentors because they one day can start abusing their access.

And admins, because they can also abuse their admin tools. 

Role is only available to whitelisted people. 

If it were public I'd share your concerns but its only available to a very select group of people.

And the role is moving away from being untargettable. as Kyet mentioned, dust implant is gone and will remain semi NTR level.

And you're assuming we are going to abuse the role and position AT ANY MOMENT.

Do you really have so little trust for us mentors? We aren't stupid to misuse it. And IF such abberations appear they would violate the trust put in them and the wouldn't remain mentors for much longer. Admins aren't blind.

Also really? Thats just disrespectful to both the team that picked us and us for having so little faith in us.

You can theorize such scenarios for every job and position on the station/server. everything can be abused.

---

I can't see how you see this as something THIS negative. It feels more organic to learn from someone who can actually tutor you than bouncing clunky mhelps back and forth with often little to no context.

---

As for roleplaying argument. 

You can spin RP any way you want, our lore is very loose, you can argument anything in any way.

**Comment by procdrone** [comment]
@Pixelzilla 

With spawning as normal crewman there is much much more concerns from round meta. Imagine that a civilian comes up to HoP line and asks for robotics access to teach someone. 

or to sec. Imagine HoS giving access to some random civilian to tutor their department.

Yes, thats not going to happen. Noone is going to trust them and allow them inside departments.

There would be no trust involved. 

And imagine heads letting in mentors on "i know they are mentor" basis.

Thats real meta gaming. 

It simply wouldn't work.

(I wont even mention for example when you're working in a deperartment. crew to crew training only really happens within same departments and not always. It also removes two people from a department work instead of one. Constant interruptions to do work etc....)

**Comment by datlo** [comment]
While I think the idea of a IC mentor role isnt bad, this looks like the wrong way to do it for me.

My suggestion : A new silicon role TeacherBot TM that isnt linked to the AI and doesnt have binary comms, with a lawset that prevents them from directly interfering with station matters other than teaching NT staff how to do things and whatever else you want the role to do. This explains ICly why the mentors wont interfere with antags/security/etc, and means that any OOC issues rulewise are already covered by silicon policy in server rules. This also gives the mentors all access, again, with IC justification : its a different type of cyborg.

**Comment by MRgaty** [comment]
Would rather give mentors other OOC tools like aghost than giving them a whole new role that is not really allowed to get involved in the flow of the round.

**Comment by Regular-Joe-SF** [comment]
I like the idea. For example if there is something at medbay they cannot do, it would fit the immersion quite better in my opinion that a _surgeon_ gets the details for a transplant surgery from an instructor, than from somebody _chemist_ NN, small example from yesterday.

Ideally, of course, they would get help from a fellow more experienced doctor or the CMO, but shifts happen where nobody IC can aid, or have time to do so (referring to what Farie wrote above). A special job creates action designed for that job, when it comes to the question why couldn't a mentor or, say, anybody else that has the knowledge act as helper. It kind of breaks the immersion rather that somebody assistant or otherwise not in the  job in question that shift starts to teach things. That action is easy to feel and get regarded as besserwisser, too.

Then, one can ask and mentors can help with the OOC tools. But not everything ends up to there, one could do not-optimal or bad procedures without realising and asking that via the mhelp. Happens with SR revivals, brigging procedures and such. The instructor could tackle that, with an IC immersionable way.

That leads, though, to the points that Ty brought forward: the job shouldn't be regarded as an authority nor they should actively take on the matters, but could this be avoided, while the instructor could still stop a single person's doings in order to let them know how it is done, if there's clear doubt of something going badly wrong, ie. a SR revival? Not sure whether the latter is what is intented, but I'd see that could happen. To be considered how to guide/limit/completely forfeit that.

When it comes to the risk that it could be regarded as a mini cc officer or such, while it would be a new thing to limit a job to admins AND mentors, I'd believe that by making it explicitly clear, that it isn't anything of a CC officer/authority/etc, would just do as countering that for the mentors playing it. To the crew, the instructors would need to give that impression by their own action, along with wiki telling what's the matter with this role.



And moreover, on Ty's points, on access to all comms including command, and avoiding the role to become a run around doing nothing one - an idea: 

how about if when joining, you would be first presented a choice that asks a department you would attach yourself? After the choice, boom, Medical Instructor NN has arrived to the station, and so the job would be about staying at single department, being present for questions or advising if there clearly is something or somebody in need of that. As the role would be a late join one, one would decide beforehand the department to head according to the need, which could be asked from mchat if there appears to be such. If no need, cryo time. While that could be done without such a mechanical aid, it would guide people to do it such as well as serve the other players at departments by letting they know they got this instructor aboard for this department. 

Downside of that idea is that perhaps there isn't that much to do in a department, and it might be more of a tentation to keep your own hands out of any action and decisionmaking, if you'd be assigned to just one department. 


The concern though, this would be kind of a new thing in many ways as already mentioned. Not a too odd thing?

File: code/game/jobs/job/central.dm [hunk 2]
```diff
@@ -104,3 +104,62 @@
...
+	glasses =  /obj/item/clothing/glasses/sunglasses
+	id = /obj/item/card/id/nanotrasen
+	belt = /obj/item/storage/belt/utility/full/multitool
+	pda = /obj/item/pda/heads/ntrep
+	implants = list(
+		/obj/item/implant/mindshield
+	)
+	backpack = /obj/item/storage/backpack/satchel
+	backpack_contents = list(
+		/obj/item/melee/classic_baton/telescopic = 1,
```
Comment (farie82) [inline code/game/jobs/job/central.dm:151]: Design: I'm not sure this is a good idea to give them. It will help against the greytide that's for sure but it might give the wrong impression. It can also be stolen

**Comment by Spacemanspark** [comment]
Wouldn't mind the idea of this if it's tweaked a bit.

But I also wouldn't mind giving mentors the ability to aghost.

**Comment by AffectedArc07** [comment]
I have been tempted to port a feature from Oracle mentors which I have missed forever. Mentors get a (F) button, but it locks you to their viewport and you see through their character (walls are still opaque etc). You cant move around from them, and you can only return to your own mob.

I think that could work here, but its down to the heads.

**Comment by McRamon** [comment]
Its weird from ic perspective for a person to be expert in every field (sure we dont enforce knowledge restrictions but still, for regular personnel its not obvious they are experts in everything)

I actually like the idea of it being a borg and not a normal crewmember, woukd make much more sense and lawset will help with making it work as intended, but i assume it would be kinda snowflakey

If it is a crewmember, it needs new clothes. Not nt officer suit. I assume the guy is in no way an officer, just a trustworthy nerd. I am willing to design his suit if needed

**Comment by farie82** [comment]
The character can be randomly generated upon spawning and be an IRC bound to NT with laws saying it should not interfere with the crew and should only be there to teach people.
Knowing NT they would create flawed laws that would not benefit them fully (look at AI laws, maint drones etc).

This would make it a bit more believable like the comments above said. And would result in less snowflake code by making another carbon or a silicon subtype that can use hands etc.

**Comment by procdrone** [comment]
@farie82 It already only have the NT official suit both BS and NTR get. Its not CC level clothing.

**Comment by Ty-Omaha** [comment]
@McRamon Naval Officers and CC officials wear different clothes now. The uniform the NTR and everyone else has is no longer the officer suit because people were saying and acting they were an officer, when they weren't and everyone always kept confusing CC officials for NTR and other station jobs. So now officers have their own special suit.



https://github.com/ParadiseSS13/Paradise/pull/12706

**Comment by Dinarzad** [comment]
I think the role of an IC, teaching focused job is fine. It is simply an additional tool in the toolbelt of Mentors to do the job they've volunteered for.

A lot of people are concerned about the job's tools though, and what might happen if they fell into the wrong hands, A Cyborg/IRC style mob could work, or, if I may make a suggestion, to split up the job a bit?
Right now, as I understand it, it is one big teaching job. One job (But not just one person) that teaches everything.  This has RP weirdness to it, sure, but I am more concerned that such mentors will be over-whelmed by questions from all sides. You have new doctors, new engineers, new atmos, maybe even new Command or Security, plus whatever is going on in actual mentorhelps.
  Additionally, most players don't know -everything-.  You have players who are damned wizards at Atmospherics, but are not as up to date on medical job knowledge for example.

So what if we split this up into departmental jobs? A teacher for Medical, Engineering, Science, etc.  This limits their tools to reduce the 'loot pinata' concerns I have seen raised, and also gives that role more of a focus.  If Medical is in full malpractice bay, then it's got a more focused department to work on then trying to do the entire station at one time.  We get a LOT of players, and Mentors are only human, there's only so much screaming and questions and data someone can effectively process at one time.

File: code/game/jobs/job/central.dm [hunk 3]
```diff
@@ -104,3 +104,62 @@
...
+	name = "Nanotrasen Instructor"
+	jobtype = /datum/job/ntinstructor
+
+	uniform = /obj/item/clothing/under/rank/centcom/representative // formal NT uniform
+	shoes = /obj/item/clothing/shoes/centcom // fancy shoes
+	head = /obj/item/clothing/head/beret/blue // generic blue beret
+	l_ear = /obj/item/radio/headset/heads/ntinstructor //
+	glasses =  /obj/item/clothing/glasses/sunglasses
+	id = /obj/item/card/id/nanotrasen
+	belt = /obj/item/storage/belt/utility/full/multitool
```
[the remark this answers is not in the document]
Comment (Kyep) [reply code/game/jobs/job/central.dm:144]: This was included because they're going to spend a fair amount of time teaching new civilians and engineers. This includes stuff like construction, hacking doors, etc. To that end, a toolbelt is useful. Yes, they could in theory pick one up from eng, but given how freely available toolbelts/tools are, I think it makes sense for them to have one. It isn't a very valuable item, but it makes a significant difference in their ability to teach construction.

File: code/game/jobs/job/central.dm [hunk 4]
```diff
@@ -104,3 +104,62 @@
...
+	backpack_contents = list(
+		/obj/item/melee/classic_baton/telescopic = 1,
+		/obj/item/pinpointer/crew = 1,
+		/obj/item/healthanalyzer/advanced = 1,
+		/obj/item/laser_pointer/blue = 1
+
+	)
+	box = /obj/item/storage/box/centcominstructor
+
+/datum/outfit/job/ntinstructor/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
```
[the remark this answers is not in the document]
Comment (Kyep) [reply code/game/jobs/job/central.dm:159]: I actually considered doing this. But I decided it would cause more problems than it would solve. For example, what if they die and are cloned? They can't re-equip their old items. What if they're gibbed? Someone picks up their ID and cannot drop it. What if they decide there are not enough people who need instruction and they want to transfer to another job? They can't hand over their ID for job transfer. Etc.

File: code/game/objects/items/devices/radio/encryptionkey.dm [hunk 5]
```diff
@@ -120,6 +120,11 @@
 	icon_state = "com_cypherkey"
 	channels = list("Command" = 1, "Security" = 0, "Engineering" = 0, "Science" = 0, "Medical" = 0, "Supply" = 0, "Service" = 0)
 
+/obj/item/encryptionkey/heads/ntinstructor
+	name = "Nanotrasen Instructor's Encryption Key"
+	icon_state = "com_cypherkey"
+	channels = list("Command" = 1, "Security" = 1, "Engineering" = 1, "Science" = 1, "Medical" = 1, "Supply" = 1, "Service" = 1)
```
[the remark this answers is not in the document]
Comment (Kyep) [reply code/game/objects/items/devices/radio/encryptionkey.dm:126]: Changed

File: code/game/jobs/job/central.dm [hunk 6]
```diff
@@ -104,3 +104,62 @@
...
+	uniform = /obj/item/clothing/under/rank/centcom/representative // formal NT uniform
+	shoes = /obj/item/clothing/shoes/centcom // fancy shoes
+	head = /obj/item/clothing/head/beret/blue // generic blue beret
+	l_ear = /obj/item/radio/headset/heads/ntinstructor //
+	glasses =  /obj/item/clothing/glasses/sunglasses
+	id = /obj/item/card/id/nanotrasen
+	belt = /obj/item/storage/belt/utility/full/multitool
+	pda = /obj/item/pda/heads/ntrep
+	implants = list(
+		/obj/item/implant/mindshield
```
[the remark this answers is not in the document]
Comment (Kyep) [reply code/game/jobs/job/central.dm:147]: Changed the 0 values to false.
They used to have a dust implant, but I removed that after discussion with the other heads.

**Comment by Kyep** [comment]
@McRamon I'm not making it a borg unless the other heads of staff indicate they prefer that option. While borgs are appealing as they have all access, are bound to laws, and drop nothing useful on death, they still have several severe limitations. One of those being that they lack hands - and thus are completely unable to touch anything that needs hands.  Another limitation is that they're extremely inflexible. They can't, for example, use any handheld object that isn't built into their module list. So for example if they don't have a medical scanner, they can't just pick one up and use it during a medical tutorial.

@Dinarzad I'm not going to split this into departmental jobs. Getting even this merged will be difficult. Making it 5x as complicated by splitting it into 5 departmental jobs dooms the idea to failure.

Instruction manual (that they spawn with):
ntinstructor_sop

File: code/game/jobs/job/central.dm (hunk 4 above)
[the remark this answers is not in the document]
Comment (farie82) [reply code/game/jobs/job/central.dm]: The getting gibbed won't be an issue due to the antidrop being removed if it's not set on the item initialy. Aka if the item doesn't have it defined then it won't have it on dropped.

The other two points I can understand yeah. It depends on how you want the job to take form. Is it an assistant that is actually trust worthy? Or somebody from CC that is there to just teach and not interact much with the station otherwise.

In terms of cryoing and such. You can maybe make it so that if they are cryod the players are taken back to the lobby.

**Comment by Regular-Joe-SF** [comment]
At my first impression above I thought the instructor would be cool to be fixed to a single department of their choice at a time, and that it might pose a problem where to limit their interaction with crew.

But then thinking it again, after seeing this manual - as the job is intended to be about only helping if asked, or after accepted offer, it indeed is good to be one job over all the departments. The instructor per these guidelines may not actively interrupt people’s doings, so there's no point on watching just one dept, and then departmental instructors, as senior engi/doc/sec, are indeed a more complicated affair than this one job - remembering the sec instructor discussion at forums a year ago. As is, this would avoid those things, seems very good to me.

**Comment by TDSSS** [comment]
I do like the idea of it being an IPC/IRC derivate. Though I am not sure the job system would support that by default.

Speaking of, why is this even a job and not, say, a mob spawner/ghost role job?

**Comment by McRamon** [comment]
I don't like how this is looking right now.
Mhelp is way simplier than calling for instructor, and if sone really new person needs a teacher, we can approach them without being any sort of instructor.
Originally i wanted a job that adds new rp opportunities, job that is focused on teaching new members. Right now, with all the restrictions it has, it basically is "summoned OOC mentor" who can come, teach you, but basically doesnt involve himself in rp.

I do not want to play a role that, in case a terror spider appears on station, must go to cryo. (There is no way with such restrictions you could stay on station during biohazard and not involve yourself with fighting it). 
I want a role that is focused on teaching, and if that role gets culted, he then focuses on teaching cult, and if xeno appear on station, that guy, instead of arming up and rushing frontline, goes to cargo, helps them to organise emergency weapon ordering, basically helps people to organise war "behind barricade".
Like a normal crewmember, not "avatar of a mentor", but is focused on teaching people, and due to his access and tools hi could possibly have, restricted to be played by mentors or admins. Granted you have trust in your mentors. If you dont, thats dissapointing and there is no need for any version of instructor.

My idea for it would be:
1) 1 for each dep
2) full dep access, aside from head office.
3) i honestly would not even mind not having mindshield, but that must be designed really well. Antag version of a teacher. Maybe special objectives, not regular, maybe only team based antags allowed, it depends. You do trust admins to be syndicate officers, why not trust mentors to be syndicate "teachers". Or just mindshield. But its problematic, mindshielded guy with full dep access will make stealth job of tators in that dep a hell.
4) obviously a set of rules, but not as restricted as the one above. If there is noone to teach, they should be allowed to involve themselves in regular roleplay. 
Basically there is one rule, kind of obvious one:dont use your access and tools to be powergaming fuck, focus on teaching. Head of dep is leading, you are there to help those in behind who is struggling to keep up". Sounds complete enough to me.
 

Another problem with one instructor for station covering all departments, and the reason i wont play him, is that the very moment someone asks me being instructor, to come to atmospherics, i would make a clown out of myself, because i barely know how to connect pipes.

I know my opinion is not really important, if at all, but whatever.

**Comment by McRamon** [comment]
However, if this is a mentor equivalent of admin cc officer, not a role they play, but a special character to join the round when needed (in case of cc/syndi officers it is to directly influence rp, in case of instructor it would be yo teach) it , in my opinion, should be tweaked, but still not sure if needed.
Especially since mentors cant just spawn themselves in. Moments when IC help can be distributed are rare, its mostly OOC questions we answer (how to enable hotkey, what to do if target ssd, how to talk in drone channel etc).
If there was an option implemented for mentors to spawn in as nt instructors themselves without asking admins, then it might be useful, but i dont think that will ever be implemented

**Comment by taukausanake** [comment]
I just got a crazy idea. What about a helper hologram? A temporary ghost role that a player can spawn when they need assistance with something. Think like the hologram recording from iRobot but it's an AI designed to assist the person using it. Something between a pAI and the AI hologram but limited to its item that it's linked to. Once the task requested has been complete the hologram terminates itself (exits the ghost role) and the item can be used for someone or thing else. The only real issue to this would be to make sure the ghostrole player knows the topic being asked

**Comment by VeteranKyl** [comment]
> I just got a crazy idea. What about a helper hologram? A temporary ghost role that a player can spawn when they need assistance with something. Think like the hologram recording from iRobot but it's an AI designed to assist the person using it. Something between a pAI and the AI hologram but limited to its item that it's linked to. Once the task requested has been complete the hologram terminates itself (exits the ghost role) and the item can be used for someone or thing else. The only real issue to this would be to make sure the ghostrole player knows the topic being asked

I believe Goonstation actually has something similar to this called a Mentor Mouse. The mentor spawns in as a purple mouse at the person's feet, and only said person can hear what the mouse says.
